test/delivery_methods/slack_test.rb

```diff
@@ -34,4 +34,22 @@ def slack_url
 
     assert_kind_of HTTP::Response, response
   end
+
+  test "logs verbosely in debug mode" do
+    class ActiveSupport::Logger
```

Comment:
I confess, I don't know the best way to stub in this test framework. Interested in recommendations.

Reply:
I think the best solution is if make the `logger` configurable so you can define your own and then we can define one just for testing like you did here.